core/storage/email/gae_models.py

```diff
@@ -337,6 +337,9 @@ class BulkEmailModel(base_models.BaseModel):
     This model is read-only; entries cannot be modified once created. The
     id/key of instances of this model is randomly generated string of
     length 12.
+
+    The recipient IDs are not stored in this model but we store all bulk emails
+    that are sent to particular user in UserBulkEmailsModel.
```

Comment:
...in this model. But, we store all ....

particular --> a particular

Reply:
Done.